.github/workflows/pull-request.yml

```diff
@@ -136,3 +136,5 @@ jobs:
       changes-externals: ${{needs.analyze-changes.outputs.changes-externals}}
       changes-c-cpp: ${{needs.analyze-changes.outputs.changes-c-cpp}}
       changes-md: ${{needs.analyze-changes.outputs.changes-md}}
+      use-sccache: ${{github.actor != 'dependabot[bot]'}}
+      sccache-write: false
```

Comment:
Only allowing read during pull request runs. This seems to be causing builds to stall/take way longer if there are multiple concurrent PR's running.